Emby.Naming/Video/ExtraRuleType.cs

```diff
@@ -5,18 +5,23 @@ namespace Emby.Naming.Video
     public enum ExtraRuleType
     {
         /// <summary>
-        /// The suffix
+        /// Match <see cref="ExtraRule.Token"/> against a suffix in the file name.
         /// </summary>
         Suffix = 0,
 
         /// <summary>
-        /// The filename
+        /// Match <see cref="ExtraRule.Token"/> against the file name.
         /// </summary>
         Filename = 1,
 
         /// <summary>
-        /// The regex
+        /// Match <see cref="ExtraRule.Token"/> against the a regex.
```

Comment:
`the a regex`? :) choose one article please
Also help string actually doesn't make much sense IMHO - what part is matched? File name, base file name, directory name, absolute path?..

Reply:
Fixed